spec/features/admin/order_cycles/complex_editing_multiple_product_pages_spec.rb

```diff
@@ -34,7 +34,10 @@
     end
 
     scenario "select all products" do
-      check "order_cycle_incoming_exchange_0_select_all_variants"
+      # replace with scroll_to method when upgrading to Capybara >= 3.13.0
+      cb_id = "order_cycle_incoming_exchange_0_select_all_variants"
+      page.execute_script("document.getElementById('#{cb_id}').scrollIntoView()")
```

Comment:
Is this possible?

I would have thought that the "view" is the browser window and that it can still be hidden by an overlay. How did you test this?

Reply:
Hi @mkllnk - I checked it visually by removing 'headless' in `spec_helper.rb` and the behavior of scrollIntoView is [as specified](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView):

> "the top of the element will be aligned to the top of the visible area of the scrollable ancestor"

Reply:
Ah, it's going to the top like clicking on an anchor. That makes it most likely visible, yes. :smile: